templates/Web/Projects/VueDefault/.template.config/template.json

```diff
@@ -0,0 +1,94 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": ["Universal"],
+  "name": "VueDefault",
+  "groupIdentity": "wts.Proj.VueDefault",
+  "identity": "wts.Proj.VueDefault",
+  "description": "",
+  "tags": {
+    "language": "Any",
+    "type": "project",
+    "wts.type": "project",
+    "wts.projecttype": "FullStackWebApp",
+    "wts.frontendframework": "Vue",
+    "wts.backendframework": "all",
+    "wts.platform": "Web",
+    "wts.version": "1.0.0",
+    "wts.displayOrder": "1"
+  },
+  "sourceName": "wts.VueDefault",
+  "preferNameDirectory": true,
+  "PrimaryOutputs": [],
+  "symbols": {
+    "wts.userName": {
+      "type": "parameter",
+      "dataType": "string",
+      "replaces": "Param_UserName"
+    },
+    "wts.wizardVersion": {
+      "type": "parameter",
+      "dataType": "string",
+      "replaces": "Param_Wizard.Version"
+    },
+    "wts.templatesVersion": {
+      "type": "parameter",
+      "dataType": "string",
+      "replaces": "Param_Templates.Version"
+    },
+    "wts.generationProjectType": {
+      "type": "parameter",
+      "replaces": "Param_ProjectType"
+    },
+    "wts.generationFramework": {
+      "type": "parameter",
+      "replaces": "Param_Framework"
+    },
+    "wts.generationPlatform": {
+      "type": "parameter",
+      "replaces": "Param_Platform"
+    },
+    "copyrightYear": {
+      "type": "generated",
+      "generator": "now",
+      "replaces": "1976",
+      "parameters": {
+        "format": "yyyy"
+      }
+    }
+  },
+  "postActions": [
+    {
+      "description": "Add Vue Dependencies",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
+      "args": {
+        "dict": "{'bootstrap': '^4.3.1', 'core-js': '^2.6.5', 'vue': '^2.6.6', 'vue-router': '^3.0.4'}",
+        "key": "dependencies",
+        "jsonPath": "package.json"
+      },
+      "continueOnError": false
+    },
+    {
+      "description": "Add Vue Dev Dependencies",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
+      "args": {
+        "dict": "{'@vue/cli-plugin-babel': '^3.5.0', '@vue/cli-plugin-eslint': '^3.5.0', '@vue/cli-service': '^3.5.0',  'babel-eslint': '^10.0.1', 'eslint': '^5.8.0', 'eslint-plugin-vue': '^5.0.0', 'vue-template-compiler': '^2.5.21'}",
+        "key": "devDependencies",
+        "jsonPath": "package.json"
+      },
+      "continueOnError": false
+    },
+    {
+      "description": "Add Vue Build Scripts",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
+      "args": {
+        "dict": "{'start': 'concurrently \"vue-cli-service serve\" \"nodemon server/server.js\"', 'build': 'vue-cli-service build', 'lint': 'vue-cli-service lint'}",
```

Comment:
defines the build scripts to be added under the "scripts" section of the package.json.

Note this is missing the && mv build server. This will be changed once Annie's PR goes in.